example_scripts/cmems_example_plot.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Mar 22 10:35:37 2022
+example to load and plot CMEMS data. Note that the example files 
+coast_example_cmems_2020_01_01_*.nc are cropped to reduce the file size (3<lat<12 and 75<lon<85 aproximatly).
+@author: jrule
+"""
+
+#################################################
+# 1. Import libraries
+#################################################
+import coast
+import matplotlib.pyplot as plt
+
+#################################################
+# Loading  data
+#################################################
+dir_nam = "/projectsa/COAsT/CMEMS/"
+filet = "coast_example_cmems_2020_01_01_download_T.nc"
+fileuv = "coast_example_cmems_2020_01_01_download_UV.nc"
+config_t = "../config/example_cmems_grid_t.json"  # make new config file
+config_uv = "../config/example_cmems_grid_uv.json"
+
+sci_t = coast.Gridded(dir_nam + filet, config=config_t)
+sci_uv = coast.Gridded(dir_nam + fileuv, config=config_uv)
+
+
+#################################################
+# calculate currents
+#################################################
+current = abs(sci_uv.dataset.u_velocity) * abs(sci_uv.dataset.v_velocity)
```

Comment:
current = abs( complex(u,v) )
or 
current = sqrt(u^2 + v^2)

Reply:
ok, thanks. How does it work to upgrade comments? Do I need to change the file on the CMEMS branch and push the changes again, or can I change things directly within this pull request?